scanner/src/test/kotlin/scanners/scancode/ScanCodeResultParserTest.kt

```diff
@@ -778,4 +778,19 @@ class ScanCodeResultParserTest : WordSpec({
             mapUnknownIssues(summary.issues.toMutableList()) shouldBe false
         }
     }
+
+    "replaceLicenseKeys()" should {
```

Comment:
Commit message question: Does this mean ScanCode reports the exactly the same license finding twice?

Reply:
Yes, that might be, if a particular license text appears twice in the same file. Then there are two entries in `"licenses"` with the same `"key"` but different `"start_line"` / `"end_line"`.

Reply:
That would make sense, but the commit message makes it sound like it would the same license twice for the same location: 
> ScanCode may return multiple license findings for a particular file
> location with equal license keys.

Reply:
I could imagine @fviernau rather meant cases like `(A and B ) or A`, if this makes even sense in practice. Same license findings but with different lines are separated anyway.

Reply:
@KorSin you're right, that's what I meant! ..and more importantly I've observed it in practice.

@mnonnenmacher I don't recall whether the  finding was "exactly the same" but the attributes startline endline and license key were the same, as the commit message says.

Reply:
@pombredanne, listing the exact same license finding twice in ScanCode's result looks like a bug to me. Note that this is still ScanCode 3.2.1-rc2, right @fviernau? Any idea if this bug (if it is one) has been fixed in newer ScanCode versions?

Reply:
yes, 3.2.1-rc2.